android/local.properties

```diff
@@ -0,0 +1,11 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file should *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+sdk.dir=PATH_TO_SDK_HERE
```

Comment:
In case you're not aware, this will actually create that dir if you try and build this without changing it.

Reply:
I didn't know that. Thanks.